docs/CONTRIBUTING.md

```diff
@@ -133,9 +135,76 @@ Please label issues with the following labels:
 2. `D*`  issue difficulty, suggesting the level of complexity this issue has. AT MOST ONE ALLOWED.
 3. `T*`  Issue topic. MULTIPLE ALLOWED.
 
-## Releases
-
-Declaring formal releases remains the prerogative of the project maintainer(s).
+## Release Process
+
+We are aiming for a **two week** release process of the `polkadot-sdk` repository. The output of a release
```

Comment:
Is it worth mentioning/explaining the merge window? I.e. just because a PR got merged before a release, doesn't mean it made it into the release.